app/models/reports/view.rb

```diff
@@ -0,0 +1,30 @@
+module Reports
+  class View < ActiveRecord::Base
```

Comment:
`Matview` can now inherit from `View` and override `refresh` and `add_table_description `.